app/controllers/manual_reject_commission_controller.rb

```diff
@@ -1,23 +1,26 @@
 class ManualRejectCommissionController < ApplicationController
   before_action :authenticate_user!, PQUserFilter
-  before_action :load_service
 
   def reject_manual
     ao_pq = ActionOfficersPq.find(params[:id])
```

Comment:
I think it would be better here to use `find!`, which would trigger a 404 in Rails. This would result into shorter code and better visibility in Google Analytics